### DISCUSSION TITLE
Adventure map, Shipwreck, the text should be adapted when the hero does not fight the ghosts [Done]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#4239] [Ideas]
When a hero visits a Shipwreck where the ghosts have already been defeated, the text should be adapted because the hero does not fight the ghosts.
It is the same in the original game.

The beginning of the text "Upon defeating the Ghosts" should simply be removed.

In fheroes2:

![No combat with ghosts 01](https://user-images.githubusercontent.com/43583994/134166572-90ff47bd-169d-41e5-808a-1f4748f7e826.png)

Here is the save file:
[Text shipwreck 01.zip](https://github.com/ihhub/fheroes2/files/7202831/Text.shipwreck.01.zip)

Version 0.9.7 4210

**Comment by ihhub** [comment]
#4246 issue was created for this. This is definitely a bug from the original game.

**Comment by LeHerosInconnu** [comment]
This has been implemented.
I close the discussion.
